### DISCUSSION TITLE
regarding Github swags

### DISCUSSION THREAD

**Discussion by mridungupta1984** [community/community#64298] [GitHub Education]
### Hi everyone,

As per the [blog post](https://github.blog/2022-05-09-prepare-for-next-semester-with-github-global-campus-and-codespaces/), Teachers can Request swag for their classroom, but despite being a verified teacher, since last one year I am unable to do so. Please look into the matter ASAP. [@queenofcorgis]

Recent thread: https://github.com/community/Global-Campus-Teachers/discussions/192 and https://github.com/orgs/community/discussions/46694

**Comment by queenofcorgis** [comment]
Hi there,

Appreciate your patience. Please note, contacting any member of GitHub Staff via unsolicited mentions or pings  is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct ](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information. 

Let's work together to keep GitHub Community a place where people feel safe to participate by being respectful of them and their time.

**Reply by mridungupta1984** [reply]
I am really apologize for unsolicited mentions, but I had to do this as I made multiple tickets and discussion posts asking for the same, but I am getting no response. It was mentioned that it would be back by May, but unfortunately, it is passed by two or three months in the past, so I wanna know is the team still planning to get it back, or they just dropped the plan off. If it might be back soon, may you please share some teaser or timeline what is gonna be released and an estimated time about when it might be back?
